src/wp-includes/pomo/po.php

```diff
@@ -260,9 +260,17 @@ public static function export_entry( $entry ) {
 			return implode( "\n", $po );
 		}
 
+		/**
+		 * Match the beginning and ending new lines between original and translation.
+		 *
+		 * @param string|null $translation   Translation text, null if don't exist.
+		 * @return string     $original      Original text.
+		 *
+		 * @return string   Translation matching new lines in the beginning or ending from the original.
```

Comment:
```suggestion
		 * @param string|null $translation Translation text, null if don't exist.
		 * @param string      $original    Original text.
		 * @return string Translation matching new lines in the beginning or ending from the original.
```

I updated in the suggestions.
- Correct the param annotation
- Correct spacing
- Remove line break between param and return annotation

Reply:
Thank you for your suggestion.
It makes it a little less readable, but I agree that is the same code style o the rest of the class, I will apply it.

Reply:
@mukeshpanchal27 Do you know why it's failing after the committed suggestion?